ecommerce_integrations/shopify/order.py

```diff
@@ -242,46 +252,67 @@ def get_tax_account_description(tax):
 	return tax_description
 
 
-def update_taxes_with_shipping_lines(taxes, shipping_lines, setting, taxes_inclusive=False):
+def update_taxes_with_shipping_lines(tax_account_wise_data, shipping_lines, setting, items, taxes_inclusive=False):
 	"""Shipping lines represents the shipping details,
 	each such shipping detail consists of a list of tax_lines"""
 	for shipping_charge in shipping_lines:
 		if shipping_charge.get("price"):
-
 			shipping_discounts = shipping_charge.get("discount_allocations") or []
 			total_discount = sum(flt(discount.get("amount")) for discount in shipping_discounts)
 
 			shipping_taxes = shipping_charge.get("tax_lines") or []
-			total_tax = sum(flt(discount.get("price")) for discount in shipping_taxes)
+			total_tax = sum(flt(d.get("price")) for d in shipping_taxes)
 
 			shipping_charge_amount = flt(shipping_charge["price"]) - flt(total_discount)
 			if bool(taxes_inclusive):
 				shipping_charge_amount -= total_tax
 
-			taxes.append(
```

Comment:
Inserting at 0 would make more sense.

Reply:
Shipping item at the top? :thinking:

Reply:
Shipping item on top of Taxes table only. Not item table.

eg:
https://user-images.githubusercontent.com/1576808/120939785-f8f75280-c719-11eb-892c-7fefa6ac883d.png